src/content/docs/guides/python/python-basics.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Python Basics
+description: Learning about the basics of Python
+sidebar:
+    order: 3
+---
+
+We're going to start off with a couple very basic things you can do in python.
+
+## Printing
+
+Printing lets you output something like a word or some numbers and even more. The `print()` function will display the given value on screen.
+
+```python
+print("Hello, World!")
+```
+
+## Comments
+
+Comments allow you to type something out without Python thinking you're writing code. Usually you'll use comments to explain what certain methods or lines of code do. Python has 2 ways to do comments.
+
+```python
+# This is an inline comment, everything after the # on the same line will be commented out.
+# Usually you'll use these to describe what a line of code is doing.
+
+"""
```

Comment:
I would remove the block comments as they aren't used outside of docstrings which should be introduced at the same time as functions - https://peps.python.org/pep-0008/#block-comments